core/models/Operator.js

```diff
@@ -13,9 +13,15 @@ module.exports = {
         title: "Password",
         type: "string",
         password: true
+      },
+      rights: {
+        type: "json"
+      },
+      isAdministrator: {
+        type: "boolean"
       }
     },
-    required: ["email", "passwordHash"]
+    required: ["email", "passwordHash", "rights"]
```

Comment:
Please remove, it should not be required. If it stays empty (let's say, just after the creating), your checking rights system should work as expected.